iroh-blobs/src/downloader.rs

```diff
@@ -728,7 +727,9 @@ impl<G: Getter<Connection = D::Connection>, D: Dialer> Service<G, D> {
                 );
 
                 let get_state = match self.getter.get(kind, progress_sender.clone()).await {
-                    Err(_err) => {
+                    Err(err) => {
+                        // This prints a "FailureAction" which is somewhat weird, but that's all we get here.
+                        tracing::error!(?err, "failed queuing new download");
```

Comment:
we already return an error `DownloadError::DownloadFailed` which admittedly does not say a lot, but the error case is handled, so I honestly do not see what do we gain from this addition

Reply:
The idea is to log the error cause.